docker/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Builder Image
+FROM rust:1.63.0 AS builder
+
+ARG IMAGE_TAG
+ARG GIT_REPOSITORY
+
+# Set Working Directory is /app
+WORKDIR /app
+
+# Node project build
+RUN git clone --depth 1 --branch $IMAGE_TAG $GIT_REPOSITORY .
+RUN apt-get update && apt-get install -y build-essential libssl-dev libclang-dev
+RUN rustup toolchain install nightly-2022-11-14 && rustup default nightly-2022-11-14
+RUN cargo build --release --bin massa-node --bin massa-client 
```

Comment:
Why to build the executables once again although there are already built during the release job ?